python/cudf/cudf/tests/test_serialize.py

```diff
@@ -269,6 +269,28 @@ def test_serialize_string_check_buffer_sizes():
     assert expect == got
 
 
+@pytest.mark.parametrize(
+    "data",
+    [
+        {"a": cudf.Series(["a", "b", "c", "a", "c", "b"]).astype("category")},
+        {
+            "a": cudf.Series(["a", "a", "b", "b"]).astype("category"),
+            "b": cudf.Series(["b", "b", "c", "c"]).astype("category"),
+            "c": cudf.Series(["c", "c", "a", "a"]).astype("category"),
+        },
+        {
+            "a": cudf.Series(["a", None, "b", "b"]).astype("category"),
+            "b": cudf.Series(["b", "b", None, "c"]).astype("category"),
+            "c": cudf.Series(["c", "c", "a", None]).astype("category"),
+        },
+    ],
+)
+def test_serialize_categorical_columns(data):
```

Comment:
could this be moved to `test_categorical.py`

Reply:
Sounds good - is it okay if I go ahead and move the other column serialization tests to their corresponding files?